utils/releaser/src/ReleaseWorker/AbstractVerifyInitialBranchReleaseWorker.php

```diff
@@ -10,9 +10,10 @@ abstract class AbstractVerifyInitialBranchReleaseWorker extends AbstractShopsysR
 {
     /**
      * @param \PharIo\Version\Version $version
+     * @param string $initialBranchName
      * @return string
      */
-    public function getDescription(Version $version): string
+    public function getDescription(Version $version, string $initialBranchName = 'master'): string
```

Comment:
The default value should not be needed here. This is in other classes to.

Reply:
Same as previous answer